README.rst

```diff
@@ -9,24 +9,57 @@ Convert from Ironic Inspector introspection format to cardiff format. Now
 you can mungetout...
 
 
-Usage
-=====
+Dependencies
+============
 
 Requires the python `hardware <https://pypi.org/project/hardware/>`_
```

Comment:
If that's purely for running `hardware-cardiff`, we should point out that it is only included up to version 0.24.0 of the `hardware` package.